spongeauth/spongeauth/settings/utils.py

```diff
@@ -0,0 +1,47 @@
+import os.path
+
+def fetch_git_sha(path, head=None):
```

Comment:
This code looks like it's copied from Raven, and is at least missing its copyright header. Hmm.

Reply:
Yeah right, missing it. I was unsure if you need it or not, I asked the sentry dev for the method in the new sdk and he said there is no one. So the suggested to just copy it. Any other suggestions?

Reply:
Sounds good, thanks.